mkdocs.yml

```diff
@@ -19,6 +19,7 @@ nav:
       - Ontology Curator GO-style: pathways/ontology-curator-go-style.md
       - Ontology Curator OBI-style: pathways/ontology-curator-obi-style.md
       - Ontology Engineer/Developer: pathways/ontology-engineer.md
+      - Semantic Engineer: pathways/semantic-engineer.md
```

Comment:
I think there is a lot of overlap between "semantic engineer" and "ontology engineer".

Reply:
could we merge ontology engineer with semantic engineer then?

Reply:
PS I didn't over think this pathway (was focusing on curator one) so happy for it to be fully revamped as semnatic engineer and taken over as so :)

Reply:
There will always be a lot of overlap, but for me, the ontology engineer is focused on design patterns, modelling, logic, while the Semantic Software Engineer is focused on using ontologies in Software and Data pipelines.

Reply:
Ontology Engineer sounds like Ontology/Semantic Architect! (I actually don't like to use 'architect' usually; sounds very Java)